py/server/tests/test_pt_proxy.py

```diff
@@ -0,0 +1,235 @@
+                    self.assertGreaterEqual(5, ct.size)
+
+    def test_reverse(self):
+        pt_proxy = self.pt_proxy.reverse()
+        for ct, rct in zip(self.pt_proxy.target.constituent_tables, pt_proxy.target.constituent_tables):
+            self.assertEqual(
+                ct.to_string(num_rows=1),
+                rct.tail(num_rows=1).to_string(num_rows=1),
+            )
+            self.assertEqual(
+                ct.tail(num_rows=1).to_string(num_rows=1),
+                rct.to_string(num_rows=1),
+            )
+
+    def test_snapshot(self):
+        t = empty_table(10).update(
+            formulas=["Timestamp=io.deephaven.time.DateTime.now()", "X = i * i", "Y = i + i"]
+        )
+        pt_proxy = self.pt_proxy.snapshot(t, cols="a")
+        self.assertEqual(4, len(pt_proxy.target.constituent_table_columns))
+        self.assertTrue(all(ct.size == 0 for ct in pt_proxy.target.constituent_tables))
+
+    def test_sort(self):
+        sorted_pt_proxy = self.pt_proxy.sort(order_by=["a", "b"],
+                                             order=[SortDirection.DESCENDING, SortDirection.ASCENDING])
+        for ct, sorted_ct in zip(self.pt_proxy.target.constituent_tables, sorted_pt_proxy.target.constituent_tables):
+            self.assertEqual(ct.size, sorted_ct.size)
+
+    def test_sort_descending(self):
+        sorted_pt_proxy = self.pt_proxy.sort_descending(order_by=["a", "b"])
+        for ct, sorted_ct in zip(self.pt_proxy.target.constituent_tables, sorted_pt_proxy.target.constituent_tables):
+            self.assertEqual(ct.size, sorted_ct.size)
+
+    def test_where(self):
+        filtered_pt_proxy = self.pt_proxy.where(filters=["a > 10", "b < 100"])
+        for ct, filtered_ct in zip(self.pt_proxy.target.constituent_tables,
+                                   filtered_pt_proxy.target.constituent_tables):
+            self.assertLessEqual(filtered_ct.size, ct.size)
+
+    def test_where_in(self):
+        unique_table = self.test_table.head(num_rows=50).select_distinct(
+            formulas=["a", "c"]
+        )
+
+        with self.subTest("where-in filter"):
+            filtered_pt_proxy = self.pt_proxy.where_in(unique_table, cols=["c"])
+            for ct, filtered_ct in zip(self.pt_proxy.target.constituent_tables,
+                                       filtered_pt_proxy.target.constituent_tables):
+                self.assertLessEqual(filtered_ct.size, ct.size)
+
+        with self.subTest("where-not-in filter"):
+            filtered_pt_proxy2 = self.pt_proxy.where_not_in(unique_table, cols=["c"])
+            for ct, filtered_ct, filtered_ct2 in zip(self.pt_proxy.target.constituent_tables,
+                                                     filtered_pt_proxy.target.constituent_tables,
+                                                     filtered_pt_proxy2.target.constituent_tables):
+                self.assertEqual(ct.size, filtered_ct.size + filtered_ct2.size)
+
+    def test_usv(self):
```

Comment:
maybe rename `usv` to `update_select_view` since I thought this was a typo for `use`.